lib/active_stash/query_builder.rb

```diff
@@ -120,9 +120,9 @@ def between(min, max)
       private
       def set(*values)
         # Find the appropriate index to use
-        @index = @available_indexes.find do |index|
+        @index = @available_indexes.select do |index|
           index.valid_op?(@op)
-        end
+        end.sort(&index_precedence).first
```

Comment:
Is there supposed to be a `:` here? i.e. `sort(&:index_precedence)`?

Reply:
That's a great question - but it ... works!

Reply:
`&:method` works in a block where you're being passed an object with `method` on it.

`&method` is a reference to `method` in the same class that I'm in, not a reference to a method on an object in a callback.